### PR TITLE
refact: use uptime.minutes instead of uptime.seconds

### PR DESCRIPTION
**Pull request by ksw2000**
In the `libpod/info.go`, within the `uptime` anonymous struct, there's a `minutes` field, but `seconds` are being used for processing. This PR refactors this issue.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by lsm5 — APPROVED** [review]
LGTM assuming happy tests.

@containers/podman-maintainers PTAL

**Comment by lsm5** [comment]
Ignore cockpit rawhide failure. Suspect it's a mirror sync issue.

**Review by Luap99 — APPROVED** [review]
/lgtm

**Comment by giuseppe** [comment]
code LGTM, but please use your real name to sign-off the commit.

**Comment by giuseppe** [comment]
shurg... not sure what happened but my comment appeared with a delay, perhaps the patch is fine since the change is trivial and not copyrightable

**Pull request merged**
